com.github.actionless.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "com.github.actionless.oomox",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "oomox-gui",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=~/.themes:create",
+        "--filesystem=~/.icons:create",
+        "--socket=session-bus",
```

Comment:
As I mentioned this permission is still overly broad.

Reply:
but i didn't got from your previous comment what's expecte to be there

Reply:
You should run your app with the --log-session-bus argument and see which applications it tries to talk to, then add them as specific --talk endpoints.

Reply:
should i pass it to `flatpak-builder --run`?
in that case i don't have any extra output after doing so

Reply:
I mean when you run the app. 
> flatpak run --log-session-bus com.github.actionless.oomox
I suspect you can probably run it through flatpak-builder as well, but I don't tend to work that way (for no good reason)